diff --git a/engines/sherlock/image_file.cpp b/engines/sherlock/image_file.cpp
index da2dfbf5b0..6c4877a53d 100644
--- a/engines/sherlock/image_file.cpp
+++ b/engines/sherlock/image_file.cpp
@@ -124,7 +124,7 @@ void ImageFile::load(Common::SeekableReadStream &stream, bool skipPalette, bool
 
 		frame._pos = stream.pos();
 
-		if (_name.empty()) {
+		if (true) {
 			// Load data for frame and decompress it
 			frame._decoded = true;
 			byte *data1 = new byte[frame._size + 4];
